src/datachain/cache.py

```diff
@@ -24,8 +24,7 @@
 @attrs.frozen
 class UniqueId:
```

Comment:
Why don't we move to Pydantic datastructure sooner?

Reply:
I plan to do that later, as it shouldn't break compatibility.